src/task5/task1/Quadrant.java

```diff
@@ -0,0 +1,5 @@
+package task5.task1;
+
+public enum Quadrant {
+    I, II, III, IV, ORIGIN_OF_COORDINATES, THE_COORDINATE_IS_ON_THE_X_AXIS, THE_COORDINATE_IS_ON_THE_Y_AXIS;
+}
```

Comment:
please put it in class where it shoul be used

Reply:
Done